offices/_vars-offices.html

```diff
@@ -2,9 +2,6 @@
 {% set query = queries.initiative %}
 {% set initiatives = query.search_with_url_arguments(size=6, filter_has_parent="false", filter_related_office=office.slug) %}
 
-{% set tag = "foia" %}
-{% set activity_type = ["cfpb_report"] %}
```

Comment:
The activity type for the FOIA page is being set in the /offices/foia-requests/index.html file. I don't know what the decision is on whether or not we're using them in Offices. It was just in there. Also, @KimberlyMunoz and I discussed moving the activity_type variable to the cms so we don't have to create an entire folder/file just to set it.